shared/settings/settings.py

```diff
@@ -57,6 +57,13 @@
 DIFFGRAM_AWS_ACCESS_KEY_SECRET = os.environ.get('DIFFGRAM_AWS_ACCESS_KEY_SECRET')
 IS_DIFFGRAM_S3_V4_SIGNATURE = env_adapter.bool(os.getenv('IS_DIFFGRAM_S3_V4_SIGNATURE', False))
 
+# Minio Settings
+DIFFGRAM_MINIO_ENDPOINT_URL = os.getenv('DIFFGRAM_MINIO_ENDPOINT_URL')
+DIFFGRAM_MINIO_ACCESS_KEY_ID = os.getenv('DIFFGRAM_MINIO_ACCESS_KEY_ID')
+DIFFGRAM_MINIO_ACCESS_KEY_SECRET = os.getenv('DIFFGRAM_MINIO_ACCESS_KEY_SECRET')
+DIFFGRAM_MINIO_DISABLED_SSL_VERIFY = os.getenv('DIFFGRAM_MINIO_DISABLED_SSL_VERIFY', None)
+DIFFGRAM_MINIO_DISABLED_SSL_VERIFY = False if DIFFGRAM_MINIO_DISABLED_SSL_VERIFY else None
```

Comment:
This condition seems to be returning `DIFFGRAM_MINIO_DISABLED_SSL_VERIFY` to False after setting it to `True`. Maybe line 66 can be removed and you can use `env_adapter.bool(` as in line 58

Reply:
`env_adapter.bool()` is added